website/pages/docs/plugins/sources/postgresql/tables.md

```diff
@@ -0,0 +1,5 @@
+# Source Plugin: PostgreSQL
+
+## Tables
```

Comment:
Maybe we should remove this file entirely, it doesn't really make sense for dynamic source plugins